isis/src/socet/apps/socetframesettings/socetframesettings.cpp

```diff
@@ -593,7 +979,8 @@ void getCamPosOPK(Spice &spice, QString spacecraftName, double et, Camera *cam,
 
   // Return the transpose of the isisFocalPlane2SocetPlate matrix for the FrameOffAxis sensor model
   xpose_c(isisFocalPlane2SocetPlate, isisFocalPlane2SocetPlateTranspose);
-  
+
   return;
 }
+int noop_intreturn() { return (int) IVALID_MAX4; } // Eliminate compiler warning about -Wunused-variable
```

Comment:
Why is this here? Adding a no-op method to eliminate a compiler warning seems odd.

Reply:
Well, I guess I am an odd person then (I've been called worse;-); I don't like noise that needs to be ignored when compiling.

Reply:
I'm mainly concerned about suppressing a warning that is signalling something we should worry about. I'm going to remove this and see what the exact warning is. If IVALID_MAX4 is being defined somewhere and then not used, I don't think this is the correct place to address it.